src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js

```diff
@@ -0,0 +1,594 @@
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on ParagraphButton {
+          fieldButtonLabel
+          fieldButtonLink {
+            uri
+            title
+          }
+        }
+      }
+    }
+    fieldClpWhatYouCanDoHeader
+    fieldClpWhatYouCanDoIntro
+    fieldClpWhatYouCanDoPromos {
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on BlockContentPromo {
+          fieldImage {
+            entity {
+              ... on Media {
+                name
+                thumbnail {
+                  height
+                  width
+                  url
+                  targetId
+                  alt
+                  title
+                }
+              }
+            }
+          }
+          fieldInstructions
+          fieldOwner {
+            entity {
+              ... on TaxonomyTermAdministration {
+                fieldAcronym
+                fieldDescription
+                fieldEmailUpdatesLinkText
+                fieldEmailUpdatesUrl
+                fieldIntroText
+                fieldLink {
+                  uri
+                  title
+                }
+                fieldSocialMediaLinks {
+                  platform
+                  value
+                  platformValues
+                }
+              }
+            }
+          }
+          fieldPromoLink {
+            entity {
+              entityType
+              entityBundle
+              entityId
+              ... on ParagraphLinkTeaser {
+                fieldLink {
+                  uri
+                  title
+                }
+                fieldLinkSummary
+              }
+            }
+          }
+        }
+      }
+    }
+    fieldClpWhyThisMatters
+    fieldHeroBlurb
+    fieldHeroImage {
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on MediaImage {
+          image {
+            height
+            width
+            url
+            targetId
+            alt
+            title
+          }
+        }
+      }
+    }
+    fieldMedia {
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on MediaVideo {
+          name
+          thumbnail {
+            height
+            width
+            url
+            targetId
+            alt
+            title
+          }
+        }
+      }
+    }
+    fieldPrimaryCallToAction {
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on ParagraphButton {
+          fieldButtonLabel
+          fieldButtonLink {
+            uri
+            title
+          }
+        }
+      }
+    }
+    fieldSecondaryCallToAction {
+      entity {
+        entityType
+        entityBundle
+        entityId
+        ... on ParagraphButton {
+          fieldButtonLabel
+          fieldButtonLink {
+            uri
+            title
+          }
+        }
+      }
+    }
+  }
```

Comment:
I think many of these properties could be replaced by including the paragraph fragments instead.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/15648#discussion_r557608725 totally 😎